src/style.css

```
@@ -0,0 +1,92 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable max-line-length */
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.button {
+  z-index: 1;
+  border: none;
+  background: none;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  padding: 0;
+}
+
+.image {
+  width: 19px;
+  height: 19px;
+}
+
+.small {
+  width: 11px;
+  height: 11;
+  position: absolute;
+  top: -16px;
+  left: -13px;
+}
+
+.search {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 22px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+
+}
+
+.bigSearch {
+  padding: 0 0 0 62px;
+  height: 70px;
+  background-size: 19px;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+::-webkit-input-placeholder {
```

why do you need webkit?
```suggestion
.search::placeholder {
```